src/main/java/ru/test/restservice/api/CompileController.java

```diff
@@ -62,10 +64,11 @@ private String compileTexFile(){
         return message.toString();
     }
 
+    // Возврат JSON с ошибкой компиляции или pdf
```

Comment:
Пиши [JavaDocs](https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html): типа такого:
```suggestion
    /** Возврат JSON с ошибкой компиляции или pdf */
```
Потом сможешь нажать кнопочку и получить доку по всему проекту, видеть описания при листании методов в подсказках, писать в них ссылки на другие куски кода и всякие удобные штуки. Стоит почитать короче